pwndbg/commands/__init__.py

```diff
@@ -29,6 +29,7 @@ class CommandCategory(str, Enum):
     STACK = "Stack"
     REGISTER = "Register"
     PROCESS = "Process"
+    LINUX = "Linux/libc/ELF"
```

Comment:
Not sure if there's a better name for this. I feel like it may be better to group these commands together like this instead of creating too many command categories. @disconnect3d let me know what you think.

Reply:
:+1: works for me